src/services/htmlStorage.service.ts

```diff
@@ -26,6 +26,12 @@ export class HtmlStorageService implements StorageService {
         if (vaultTimeoutAction == null) {
             await this.save(ConstantsService.vaultTimeoutActionKey, 'lock');
         }
+
+        // Default theme to match the browser if the theme isn't set
+        const theme = await this.get<string>(ConstantsService.themeKey);
+        if (theme == null) {
+            await this.save(ConstantsService.themeKey, 'theme_defaultSet');
```

Comment:
This should also not be necessary once you make default theme == null. It'll be null by default.